egs/wsj/s5/steps/durmod/durmodel_prepare_examples.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
I would recommend renaming this file to get_durmodel_egs.sh just for the sake of consistency with egs preparing scripts.